ivy/functional/frontends/numpy/manipulation_routines/changing_number_of_dimensions.py

```diff
@@ -0,0 +1,31 @@
+# global
+from typing import Any, Sequence
+
+import ivy
+
+
+# squeeze
+def squeeze(
+    x: Sequence[Any],
+    axis: int,
+    /,
```

Comment:
please remove arguments from this line onwards and keep only `x` and `axis` because these are the only 2 args according to the numpy docs